workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -43,17 +41,14 @@ async def preprocess(self) -> None:
         curr_year_start = "{0}-01-01".format(latest_ohsome_stamp.year)
         curr_year_range = "{0}/{1}".format(curr_year_start, self.end)
 
-        response = await ohsome_client.query(
-            layer=self.layer,
-            bpolys=self.feature.geometry,
-        )
+        response = await ohsome_client.query(self.layer, self.feature.geometry)
         self.element_count = response["result"][0]["value"]
         self.result.timestamp_osm = dateutil.parser.isoparse(
             response["result"][0]["timestamp"]
         )
         response_contributions = await ohsome_client.query(
-            layer=self.layer,
-            bpolys=self.feature.geometry,
+            self.layer,
+            self.feature.geometry,
```

Comment:
Because the `__init__` function of the Indicator BaseClass takes only two arguments. Both have to be given. They do not have a default (argument vs keyword argument). Since there are always the same and only two arguments for every indicator I thought those parameter-keys does not serve any purpose.
```
class BaseIndicator(metaclass=ABCMeta):
    """The base class of every indicator."""

    def __init__(
        self,
        layer: Layer,
        feature: Feature,
    ) -> None:
```

Reply:
Okay if we don't want to use keys, we should be consistent of the order, I think. Just browse through the repository and you'll find cases with `layer=layer, feature=feature` and `feature, layer` (mostly in tests).

Reply:
I think in this PR all cases of `layer=layer, feature=feature` has been changed to `feature, layer`. I could not find any occurrences of the first one.
Since it has been already changed I would suggest to keep it in the PR to save time refactoring those changes out.